### PR TITLE
docs: consolidate READMEs and use Helm chart README for installation

### PR DESCRIPTION
**Pull request by ivanstanev**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [x] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,186 +5,26 @@
...
-The choice of which deployment to use depends on the permissions you have on your cluster.
-
-For _cluster_-scoped deployment you can create the necessary `ServiceAccount`, `ClusterRole`, and `ClusterRoleBinding` required for the monitor's deployment.
-These objects ensure the monitor has the right (limited) level of access to resources in the cluster. The command is as follows:
-```shell
-kubectl apply -f snyk-monitor-cluster-permissions.yaml
-```
-Note that even though the monitor operates in the whole cluster, the `ClusterRole` ensures it can only _read_ or _watch_ resources; the monitor can never modify your objects!
-
-For a _namespaced_ deployment you can create the necessary `ServiceAccount`, `Role`, and `RoleBinding` required for the monitor's deployment:
```
Comment (kat1906) [inline README.md:114]: I dont think we have this section in the helm docs

Comment (ivanstanev) [reply README.md:114]: The first part is a good one to keep, but the second is kind of handled automatically by the Helm chart so no need to create them manually

Comment (kat1906) [reply README.md:114]: ah ok, we just need to have a section on namespaced deployment in the helm README as I think it is missing

File: snyk-monitor/README.md [hunk 2]
```diff
@@ -105,20 +109,37 @@ Add the latest version of Snyk's Helm repo:
...
 
 **Please note that `/` in cluster name is disallowed. Any `/` in cluster names will be removed.**
 
+### Installation and monitoring of a single namespace
+
+The Snyk monitor can be configured to monitor only the namespace in which it is installed instead of the whole cluster:
+
+```shell
+helm upgrade --install snyk-monitor snyk-charts/snyk-monitor \
+  --namespace snyk-monitor \
```
[the remark this answers is not in the document]
Comment (ivanstanev) [reply snyk-monitor/README.md:135]: Yeah, should I reword the note right above it?

[the remark this answers is not in the document]
Comment (kat1906) [reply snyk-monitor/README.md:135]: how do you tell it which namespace to monitor? I don't understand 😆

[the remark this answers is not in the document]
Comment (ivanstanev) [reply snyk-monitor/README.md:135]: It monitors the namespace in which it is installed

``` bash
--namespace snyk-monitor # <--- Install in this namespace
--set scope=Namespaced # <--- Tell it to scan only this namespace and not the whole cluster
```

[the remark this answers is not in the document]
Comment (kat1906) [reply snyk-monitor/README.md:135]: ok that is what I thought 😆 just a bit confusing. Maybe change to something like: `--namespace some-namespace-to-be-monitored` or something 🤷

**Review by kat1906 — APPROVED** [review]

**Pull request merged**